MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
Ha, 3 possibilities! I'm fairly sure I remember seeing internal field == Pascal case mentioned in an MS style guide some years ago, but I have no source to back me up. It's what ReSharper suggests... if I'm allowed to use that as a source :)